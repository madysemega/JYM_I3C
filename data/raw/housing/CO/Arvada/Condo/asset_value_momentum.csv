predictions_asset_value_momentum
-0.08349085459994048
0.09395434399833939
0.1393479253308333
-0.18034596687373805
0.007287104013796274
-0.02764450049400064
-0.09218170899104605
0.15141181731092265
0.24622644138721617
-0.0785265469982683
-0.08146301802831317
0.010772504660452667
-0.08062066325781875
-0.06911194913027588
-0.22247423746281914
0.09748034765965748
-0.06745049691193393
0.0809138338075634
-0.14446525111985045
-0.09703223348142404
0.2313924560669534
0.1069384096873862
-0.11857251932960527
-0.08538606868222719
0.12051679411435678
0.06779535741350437
-0.2679558936765986
