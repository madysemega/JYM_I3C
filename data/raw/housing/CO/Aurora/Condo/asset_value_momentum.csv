predictions_asset_value_momentum
-0.11011410020170624
-0.14867873323690858
-0.2290694178383944
0.3678809965492135
-0.33394114007838105
0.4844315077184216
0.46025443414679046
0.28549610784549945
0.19348108185362092
-0.29293145102687435
-0.24414858155272193
-0.3985592271898365
-0.3823836302705598
-0.1668904161378788
0.46910626742116224
0.20970320567378406
0.4857056458584495
0.27063970796508413
0.11722223144821053
0.22510070672881294
0.4818934855422579
-0.387081434034235
0.25461767063368723
0.06106019768451393
0.384574590043629
0.23723775331381033
0.12222671617807726
