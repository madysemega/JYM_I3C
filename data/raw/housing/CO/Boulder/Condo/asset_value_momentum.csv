predictions_asset_value_momentum
0.04020644425894065
-0.3346814809684873
0.040935752256935624
-0.1322047044880199
-0.10295840573495013
-0.005273209465046437
-0.2176161824260524
-0.025738569159894475
-0.2399482545774907
-0.14817884372504916
-0.12504882709966622
-0.0739029040217101
-0.24271522376537047
-0.15900862707328026
-0.1510159788878546
-0.17423265155605147
0.10630055451113411
-0.2049648805518145
-0.2643061941917706
-0.11319159485695102
-0.3166028231761519
-0.13077052362024755
0.041366598548962935
-0.17300353914297828
-0.0862691562040013
-0.39470238467587193
-0.16930913069326878
