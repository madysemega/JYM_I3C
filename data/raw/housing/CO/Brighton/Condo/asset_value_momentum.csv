predictions_asset_value_momentum
0.4273155860299643
-1.1373043242942473
-0.16640381210130867
-0.10744615662348828
-0.8115183055007734
-0.5280826874982889
0.9654202934167936
-0.1099574591503258
-0.09471161598636495
-0.2749828815296759
-0.630063350826116
-0.7025460825832998
-1.0549840360287095
-1.0953282850090107
0.4836213327835613
0.2634238447758109
0.5076899404069113
-0.9609867473508863
0.12660628087954715
-0.21399357128068527
-0.6287589763718714
0.3274592401008164
-1.0476215702133558
0.23820364345634093
0.2862469549189814
-1.5303303297831155
0.7825416261499932
