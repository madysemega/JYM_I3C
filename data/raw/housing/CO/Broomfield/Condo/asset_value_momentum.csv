predictions_asset_value_momentum
0.04219795976480121
-0.11359010614063458
0.16186035824029585
0.11471057393678993
0.0832234781035977
-0.10428003142374992
0.0747471283955074
-0.11594747068265482
-0.051218780420772964
0.05991476313789997
-0.06731625761413654
-0.014942668113574626
-0.26367369172598304
0.08485302748204789
0.24481947855845174
-0.23040878119099067
0.046549836536262
-0.01478721849966428
0.2003064602426831
-0.06242077026570834
0.11859360840341845
-0.061400480785590544
0.1625367279169525
0.00977805938544256
-0.3324224389308245
-0.15111350212241537
0.37697017325099824
