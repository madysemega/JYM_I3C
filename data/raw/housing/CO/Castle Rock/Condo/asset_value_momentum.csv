predictions_asset_value_momentum
0.28071632868467417
0.22180071966922
0.42352693277831543
-0.06553347294029654
0.4186136216220047
-0.03925479293234104
-0.05012100969666504
-0.13232165792897935
0.22699779987279603
-0.09535671731067741
0.03370806889350224
-0.11757094015304158
0.00044141281872537766
-0.004312316081185634
-0.08934791490313261
0.12750512864469288
0.95423220639866
-0.19073749316600633
-0.07780728867692063
-0.18527692223542816
-0.2586587472201423
0.06975170791656236
0.17203708456121253
-0.05257135717205195
0.02615876725760399
0.44589635725015675
-0.1172632767771079
