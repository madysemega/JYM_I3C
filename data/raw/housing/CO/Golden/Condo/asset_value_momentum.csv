predictions_asset_value_momentum
-0.34129837016737125
-0.05912710169611734
0.31553314936562094
0.5995813925937982
-0.27793900508051256
0.4251557721611567
0.272186735292319
0.556196016778898
-0.568637167395252
-0.3135872839452612
0.11769366560450817
-0.26095485317131234
-0.6264469134593948
-0.3820904205490612
0.6867688157584418
0.8486552331432077
0.503392568727188
-0.030593389264785026
-0.20484415816514379
-0.2535138511681604
-0.7169133625126084
-0.3346264815168576
-0.3272812882462544
-0.06080636228510124
0.07859241429823505
-0.6239805530536271
0.1347749122695129
